src/styles/sidebar/components/NotebookView.scss

```diff
@@ -4,18 +4,21 @@
 
 .NotebookView {
   display: grid;
-  row-gap: var.$layout-space--xsmall;
+  row-gap: var.$layout-space--small;
   grid-template-areas:
     'heading'
+    'filters'
     'results'
     'items';
 
   &__heading {
+    line-height: 1;
```

Comment:
This was needed to normalize the row height so that there wasn't dead space above and below the heading text.